spec/controllers/trigger_controller_spec.rb

```diff
@@ -62,5 +62,30 @@
 		  		expect(response).to redirect_to(trigger_path(other_new_trigger.id))
 			end
 		end
+		
```

Comment:
These tests look great, but they are really model tests. I would move it to the spec for trigger model! Also make sure the indentation is consistent.